src/Tribe/Template.php

```diff
@@ -646,23 +655,7 @@ public function template( $name, $context = [], $echo = true ) {
 		 */
 		do_action( "tribe_template_before_include:$hook_name", $file, $name, $this );
 
-		// Only do this if really needed (by default it wont).
-		if ( true === $this->template_context_extract && ! empty( $this->context ) ) {
-			// We don't allow Extracting of a variable called $name
-			if ( isset( $this->context['name'] ) ) {
-				unset( $this->context['name'] );
-			}
-
-			// We don't allow the extraction of a variable called `$file`.
-			if ( isset( $this->context['file'] ) ) {
-				unset( $this->context['file'] );
-			}
-
-			// Make any provided variables available in the template variable scope.
-			extract( $this->context ); // @phpcs:ignore
-		}
-
-		include $file;
+		$this->template_safe_include( $file );
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: a47269d8a75e1285488ba35e3d8a1dc5</sub>